cts/scheduler/xml/promoted-ordering.xml

```diff
@@ -242,24 +242,24 @@
           <expression attribute="#uname" operation="eq" value="webcluster01" id="expression.id23076"/>
         </rule>
       </rsc_location>
-      <rsc_location id="run_intip_1_promoted" rsc="intip_1_master">
+      <rsc_location id="run_intip_1_promoted" rsc="intip_1_active">
```

Comment:
We left this ID as "promoted" and changed the others to "active". We changed "unpromoted" to "passive" below. Doesn't matter just inconsistent.

Similar for `<meta_attributes id="primitive-intip_1_promoted.meta">`

Reply:
Ah, I missed that because I was just looking for the old name. Not worth changing, especially since it was already different from the resource name ...